src/components/data-exploration/embedding/Embedding.jsx

```diff
@@ -22,12 +22,13 @@ import {
   clearPleaseWait,
   renderCellSetColors,
   colorByGeneExpression,
+  colorInterpolator,
 } from '../../../utils/embeddingPlotHelperFunctions/helpers';
-import legend from '../../../../public/static/media/viridis.png';
 import { isBrowser } from '../../../utils/environment';
 import PlatformError from '../../PlatformError';
 
 import { loadProcessingSettings } from '../../../redux/actions/experimentSettings';
+import * as d3 from 'd3-scale';
```

Comment:
What do we use this package for?

Reply:
It's not a new package but it is a new import in this file; yes, it is to generate the min and max colours from the scheme so that we can display them as a gradient in the legend below.

Reply:
So you didn't find the colour scheme we want from the available ones provided by Vega - https://vega.github.io/vega/docs/schemes/ ?